webrender/src/device/gfx/program.rs

```diff
@@ -407,6 +406,7 @@ impl<B: hal::Backend> Program<B> {
         image: &ImageCore<B>,
         binding: &'static str,
         cmd_buffer: &mut hal::command::CommandBuffer<B, hal::Graphics>,
+        sampler: Option<&B::Sampler>,
     ) {
         if let Some(binding) = self.bindings_map.get(&("t".to_owned() + binding)) {
```

Comment:
we shouldn't be heap allocating ("to_owned") here on every move. A we shouldn't even string compare here, it's a relatively hot spot.